pkg/web/lastOperation.go

```diff
@@ -0,0 +1,62 @@
+package web
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/feedhenry/negotiator/pkg/deploy"
+	"github.com/feedhenry/negotiator/pkg/log"
+	"github.com/feedhenry/negotiator/pkg/status"
+	"github.com/gorilla/mux"
+)
+
+type StatusRetriever interface {
```

Comment:
Can we get some comments on all the public stuff in this file for future reference.